src/read.rs

```diff
@@ -193,7 +191,7 @@ impl<R> ReadExt for R
 
     #[inline(always)]
     fn read_le_f32(&mut self) -> io::Result<f32> {
-        self.read_le_u32().map(|u| unsafe { mem::transmute(u) })
+        self.read_le_u32().map(|u| unsafe { mem::transmute::<u32, f32>(u) })
```

Comment:
There is also [`from_le_bytes`](https://doc.rust-lang.org/std/primitive.f32.html#method.from_le_bytes) since Rust 1.40. If you are concerned about the `unsafe`, and we are bumping the minimum supported Rust version to 1.36 for the `MaybeUninit` anyway, then it’s a small step to go to 1.40, and then we can use `from_le_bytes` here, and get rid of the `unsafe` and `transmute` entirely.